hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNode.java

```diff
@@ -681,17 +681,23 @@ public boolean isSecurityEnabled() {
   }
 
   private void stopCommonServices() {
+    if (leaderElection != null && leaderElection.isRunning()) {
+      try {
+        leaderElection.stopElectionThread();
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
use the logger If you want to print something.

Reply:
done